src/style.css

```diff
@@ -0,0 +1,106 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url(./fonts/Avenir-Book.ttf) format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 600;
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf) format("truetype");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+  width: 100%;
+  height: 70px;
+}
+
+.search-form--small {
+  height: 42px;
+}
+
+.search-form__label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search-form__label:after {
+  position: absolute;
+  content: "";
+  display: block;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
```

Comment:
I assume these values are not halfs of the height and padding to counter :after 19x19 size. This will be hard to work with as you have to manually calculate images position. A better alternative would be to position base of the :after with half of these values and then translate it by 50% of the images size -> this way you will more accurately position it in the center.
```css
top: 35px;
left: 31px;
transform: translate(-50%, -50%) 
```
or you could even use some variables as these values are dependent on each other
```css
.search-form {
  height: var(--big-form-height);
  ...
}

.search-form__label:after {
  top: calc(var(--big-form-height) / 2);
  left: calc(var(--big-input-padding-l) / 2);  /* -> same for the padding */
  transform: translate(-50%, -50%);
  ...
}
```